### PR TITLE
Add support to attach files in email using email sink

### PR DESCRIPTION
**Pull request by minudika**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Review by NisalaNiroshana — APPROVED** [review]

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java [hunk 1]
```diff
@@ -192,6 +201,34 @@
                                 + "bcc='bcc1.account@gmail.com"
                                 + ")"
                                 + "define stream fooStream (name string, age int, country string);"),
+
+                @Example(description = "Following example illustrates how to publish events using the"
+                        + "email sink. Here files are also attached to the email."
```
Comment (dilini-muthumala) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java:206]: Need space between "the" and "email" :)

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java [hunk 2]
```diff
@@ -127,11 +130,18 @@
                                 + " then addresses can be given as a comma separated list.",
                            type = DataType.STRING,
                            optional = true,
-                           defaultValue = "None")
-
+                           defaultValue = "None"),
+                @Parameter(name = "attachments",
+                        description = "File paths of the files that needs to be attached to the email.\n" +
+                                "These paths should be absolute paths.\n" +
+                                "They can be either a directories or files. If it's a directory all the files inside " +
```
Comment (grainier) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java:137]: Should be "They can be either directories or files".

Comment (minudika) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java:137]: fixed by 3d4e83b2f5d203578f81e8084b33d5770a6d0b18

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java (hunk 1 above)
Comment (minudika) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java:206]: fixed by 3d4e83b2f5d203578f81e8084b33d5770a6d0b18

**Pull request merged**